pkgs/applications/networking/mailreaders/alot/default.nix

```diff
@@ -0,0 +1,64 @@
+{ stdenv, lib
+, pythonPackages, fetchFromGitHub, notmuch, file, gpgme
+, gnupg ? null
+, withManpage ? true }:
+
+with pythonPackages;
+
+buildPythonApplication rec {
+  pname = "alot";
+  version = "0.7";
+  outputs = [ "out" "man" ];
+
+  disabled = isPy3k;
+
+  src = fetchFromGitHub {
+    owner = "pazz";
+    repo = "alot";
+    rev = "${version}";
+    sha256 = "1y932smng7qx7ybmqw4qh75b0lv9imfs5ak9fd0qhysij8kpmdhi";
+  };
+
+  postPatch = ''
+    substituteInPlace alot/defaults/alot.rc.spec \
+      --replace "themes_dir = string(default=None)" \
+                "themes_dir = string(default='$out/share/themes')"
+  '';
+
+  checkInputs =  [ mock gnupg ];
+  nativeBuildInputs = stdenv.lib.optionals withManpage [ sphinx ];
+
+  propagatedBuildInputs = [
+    notmuch
+    urwid
+    urwidtrees
+    twisted
+    python_magic
+    configobj
+    service-identity
+    gpg
+  ];
+
+  # 3 tests fail, some unicode
+  doCheck = false;
+  postBuild = "make -C docs man";
+
+  postInstall = ''
+    mkdir -p $out/share/applications $out/man
+    cp -r docs/build/man $out/man
+    cp -r extra/themes $out/share
+
+    sed "s,/usr/bin,$out/bin,g" extra/alot.desktop > $out/share/applications/alot.desktop
+    wrapProgram $out/bin/alot \
+      --prefix LD_LIBRARY_PATH : '${lib.makeLibraryPath [ notmuch file gpgme ]}'
```

Comment:
these should be patched in the code

Reply:
leftover from the actual alot code. it seems to work fine without it so I removed it